acmeproxy/proxy/tests/test_commands.py

```diff
@@ -0,0 +1,125 @@
+from io import StringIO
+
+import pytest
+from django.core.management import call_command
+from django.core.management.base import CommandError
+
+from acmeproxy.proxy.models import Authorisation, Response
+
+
+@pytest.mark.django_db
+class TestDeleteAuthorisation:
+    def test_successful_delete(self):
+        Authorisation.objects.create(
+            name="example.com", secret="test_secret", created_by_ip="127.0.0.1"
+        )
+        out = StringIO()
+        call_command("deleteauthorisation", "example.com", stdout=out)
+        assert Authorisation.objects.count() == 0
+
+    def test_different_name(self):
```

Comment:
Is this maybe, more accurately, asserting the behaviour when the domain doesn't have an auth record? Maybe a different test name would be more clear?

```suggestion
    def test_no_domain_matched(self):
```